src/Microsoft.ML.GenAI.Phi/Phi3/Phi3ChatTemplateBuilder.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AutoGen.Core;
+using Microsoft.Extensions.AI;
+using Microsoft.ML.GenAI.Core;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using TextContent = Microsoft.SemanticKernel.TextContent;
+
+namespace Microsoft.ML.GenAI.Phi;
+
+public class Phi3ChatTemplateBuilder : IChatTemplateBuilder, IMEAIChatTemplateBuilder
+{
+    private const char Newline = '\n';
+
+    public static Phi3ChatTemplateBuilder Instance => new Phi3ChatTemplateBuilder();
```

Comment:
Nit: Properties like this are expected to return the same instance on each call. If this is indeed supposed to be creating a new instance on each access, it should be a method.

Reply:
I've updated it to `public static Phi3ChatTemplateBuilder Instance {get;} => new Phi3ChatTemplateBuilder();`